pkg/prometheus/server/statefulset_test.go

```diff
@@ -1289,9 +1179,7 @@ func TestThanosObjectStorageFile(t *testing.T) {
 				}
 			}
 		}
-		if !found {
-			t.Fatalf("--tsdb.path argument should be given to the Thanos sidecar, got %q", strings.Join(sset.Spec.Template.Spec.Containers[3].Args, " "))
-		}
+		require.True(t, found, "--tsdb.path argument should be given to the Thanos sidecar, got %q", strings.Join(sset.Spec.Template.Spec.Containers[3].Args, " "))
```

Comment:
I checked why the tests are failing and I got this error

```
--- FAIL: TestThanosObjectStorageFile (0.00s)
panic: runtime error: index out of range [3] with length 3 [recovered]
        panic: runtime error: index out of range [3] with length 3
```
When I changed it to ```sset.Spec.Template.Spec.Containers[2].Args```, it was working fine. Wdyt about this?